docs/basics/dataframe.md

```diff
@@ -150,3 +149,42 @@ Let's try it out:
     df = pl.LazyFrame({'a': [1, 1, 2], 'b': [4, 5, 6]})
     print(func(df).collect())
     ```
+
+## Example 4: multiple inputs
```

Comment:
I hope this section exemplify the new behavior, if anyone has more ideas I am all ears